apps/design/shared/src/layout.ts

```diff
@@ -104,6 +80,58 @@ function textWrap(text: string, fontStyle: FontStyle, width: number): string[] {
   return lines;
 }
 
+function textWrap(text: string, fontStyle: FontStyle, width: number): string[] {
+  return text.split('\n').flatMap((line) => wrapLine(line, fontStyle, width));
+}
+
+function textHeight(textLines: string[], fontStyle: FontStyle): number {
+  return textLines.length * fontStyle.lineHeight;
+}
+
+function TextBlock({
```

Comment:
Added this new subcomponent to help with rendering multi-line text where the lines have different styling

Reply:
Cool, looks like this helps simplify a lot below!